src/components/vuestic-components/va-tabs/VaTabs.demo.vue

```diff
@@ -1,136 +1,283 @@
         >
           {{ title }}
         </va-tab>
+        <va-tab label="Tab without name" />
+        <va-tab
+          icon="fas-home"
+          label="Tab with icon"
+          name="icon"
+        />
+        <va-tab
+          icon="face"
+          label="Disabled tab"
+          disabled
+          name="disabled"
+        />
       </va-tabs>
     </VbCard>
     <VbCard
-      title="Centered"
-      width="400px"
+      title="Vertical Tabs"
+      style="width: 100%;"
     >
-      <va-tabs
-        center
-        v-model="tabValue"
-      >
+      <va-tabs v-model="tabValue1" vertical>
         <va-tab
-          v-for="title in tabTitles"
+          v-for="title in ['One', 'Two', 'Three', 'Four', 'Five', 'Six', 'Seven', 'Eight']"
+          :name="title"
           :key="title"
         >
           {{ title }}
         </va-tab>
+        <div style="width: 100%; height: 100%; background-color: wheat;"></div>
       </va-tabs>
     </VbCard>
+    <!--<VbCard title="Tabs With Pagination" style="width: 30%;">
+      <va-tabs v-model="tabValue1" prev-icon="arrow_back_ios" next-icon="arrow_forward_ios">
+        <va-tab
+          v-for="title in ['One', 'Two', 'Three', 'Four', 'Five', 'Six', 'Seven', 'Eight']"
+          :name="title"
+          :key="title"
+        >
+          {{ title }}
+        </va-tab>
+        <va-tab label="Tab without name" />
+        <va-tab icon="fas-home" label="Tab with icon" name="icon" />
+        <va-tab icon="face" label="Disabled tab" disabled name="disabled" />
+      </va-tabs>
+    </VbCard> -->
     <VbCard
-      title="Grow"
-      width="400px"
+      title="Tabs with names"
+      style="width: 100%;"
     >
-      <va-tabs
-        grow
-        v-model="tabValue"
-      >
+      <va-tabs v-model="tabValue2">
         <va-tab
-          v-for="title in tabTitles"
+          v-for="title in tabDynamic"
+          :name="title"
           :key="title"
         >
           {{ title }}
         </va-tab>
       </va-tabs>
+      Value: {{ tabValue2 }}
+      <div>
+        <va-button @click="tabDynamic.push(`Tab #${tabDynamic.length + 1}`)">
+          Add one more tab
+        </va-button>
+      </div>
     </VbCard>
-    <VbCard title="Overflow">
-      <va-tabs
-        style="width: 120px;"
-        v-model="tabValue"
-      >
+    <VbCard
+      title="Tabs with indices"
+      style="width: 100%;"
+    >
+      <va-tabs v-model="tabValue3">
+        <va-tab>
+          Inital tab
+        </va-tab>
         <va-tab
-          v-for="title in tabTitles"
+          v-for="title in tabAdded"
           :key="title"
         >
           {{ title }}
         </va-tab>
       </va-tabs>
+      Value: {{ tabValue3 }}
+      <div>
+        <va-button @click="tabAdded.push(`Tab #${tabAdded.length + 1}`)">
+          Add one more tab
+        </va-button>
+      </div>
     </VbCard>
-    <VbCard title="Overflow text">
+    <VbCard
+      title="Tabs with same v-model"
```

Comment:
![image](https://user-images.githubusercontent.com/5394573/78151083-dba56500-7440-11ea-8b15-422dca16601d.png)

Slider is off for newly spawned component.
If value is changed - slider has proper position.
Problem applies to vertical slider as well.